docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -49,11 +55,11 @@ unsafe
 }
 ```
 
-## Ref safety errors do not affect conversion from lambda expression to delegate
+## `ref` safety errors do not affect conversion from lambda expression to delegate
 
 ***Introduced in Visual Studio 2022 version 17.5***
 
-Ref safety errors reported in a lambda body no longer affect whether the lambda expression is convertible to a delegate type. This change can affect overload resolution.
+`ref` safety errors reported in a lambda body no longer affect whether the lambda expression is convertible to a `delegate` type. This change can affect overload resolution.
```

Comment:
When it's used in the phrase "delegate type", delegate typically isn't considered a keyword:

```suggestion
`ref` safety errors reported in a lambda body no longer affect whether the lambda expression is convertible to a delegate type. This change can affect overload resolution.
```